Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector.yml

```diff
@@ -0,0 +1,156 @@
+category: Analytics & SIEM
+commonfields:
+  id: SaasSecurityEventCollector
+  version: -1
+configuration:
+- additionalinfo: 'The instance configuration URL based on the server location: <br/>US:
```

Comment:
Is it always these 3? Lets make this a drop-down type and add a match-case in the code

Reply:
i don't think so, i am using a URL which is not one of those 3, this was taken directly from the original integration.

```Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.yml```